content/guides/use-case/wiremock.md

```diff
@@ -0,0 +1,272 @@
+
+### Prerequisites
+
+The following prerequisites are required to follow along with this how-to guide:
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+
+### Launching WireMock
+
+Launch a quick demo of WireMock by using the following steps:
+
+ 1. Clone the [GitHub repository](https://github.com/dockersamples/wiremock-node-docker) locally.
+
+    ```console
+    $ git clone https://github.com/dockersamples/wiremock-node-docker
+    ```
+
+ 2. Navigate to the `wiremock-endpoint` directory
+
+    ```console
+    $ cd wiremock-node-docker/
+    ```
+
+    WireMock acts as the mock API that your backend will communicate with to retrieve data. The mock API responses have already been created for you in the mappings directory. 
+
+ 3. Start the Compose stack by running the following command at the root of the cloned project directory
+
+    ```console
+    $ docker compose up -d
+    ```
+
+    After a moment, the application will be up and running. 
+
+    ![Diagram showing the WireMock container running on Docker Desktop ](./images/wiremock-using-docker.webp)
+
+
+    You can check the logs by selecting the **WireMock** container:
+
+    ![Diagram showing the logs of WireMock container running on Docker Desktop ](./images/wiremock-logs-docker-desktop.webp)
+
+ 4. Test the Mock API.
+
+    ```console
+    $ curl http://localhost:8080/api/v1/getWeather\?city\=Bengaluru
+    ```
+ 
+    It will return the following response:
+
+    ```plaintext
+    {"city":"Bengaluru","temperature":27.1,"conditions":"Mostly cloudy","forecasts":[{"date":"2024-09-02T07:00:00+05:30","temperature":83,"conditions":"Partly sunny w/ t-storms"},{"date":"2024-09-03T07:00:00+05:30","temperature":83,"conditions":"Thunderstorms"},{"date":"2024-09-04T07:00:00+05:30","temperature":83,"conditions":"Intermittent clouds"},{"date":"2024-09-05T07:00:00+05:30","temperature":82,"conditions":"Dreary"},{"date":"2024-09-06T07:00:00+05:30","temperature":82,"conditions":"Dreary"}]}
+    ```
+
+    You get this response due to the mapping file in the `wiremock-endpoint/mappings/getWeather/getWeatherBengaluru.json` file. 
+
+
+## Using WireMock in development
+
+   Now that you have tried WireMock, let’s use it in development and testing. In this example, you will use a sample application that has a Node.js backend. This app stack has the following configuration:
+
+  * **Local Development Environment**: The context in which the Node.js backend and WireMock are running.
+  * **Node.js Backend**: Represents the backend application that handles HTTP requests.
+  * **External AccuWeather API**: The real API from which live weather data is fetched.
+  * **WireMock**: The mock server that simulates the API responses during testing. It runs as a Docker container.
```

Comment:
Do not bold these, and do not use title case

Reply:
Removed.